src/com/lftechnology/java/training/krishna/collections/generics/AdditionDemo.java

```diff
@@ -0,0 +1,35 @@
+
+package com.lftechnology.java.training.krishna.collections.generics;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ *  Write a program in which Generic method is used to sum Integer or Double. 
+ */
+
+/**
+ * This program used to sum two numbers
+ *
+ * @author Krishna Timilsina <krishnatimilsina@lftechnology.com>
+ * @version 1.0
+ */
+public class AdditionDemo {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1118)